backend/curriculum_tracking/models.py

```diff
@@ -1010,6 +1011,10 @@ def get_teams(self, assignees_only=False):
             user_ids.extend([user.id for user in self.reviewers.all()])
         return Team.get_teams_from_user_ids(user_ids)
 
+    @property
+    def repo_url(self):
+        return self.recruit_project.link_submission or self.repository.ssh_url
```

Comment:
A link submission isn't the same as a repo_url.

Also, what if there is no recruit_project on the card? It happens...